bandit.conf

```diff
@@ -85,11 +85,11 @@
 tests:
```

Comment:
```suggestion
# tests:
```

We can comment out the tests: section as it's optional.  (If you put in `tests: []` it would likely stop running bandit tests, which is not what we want.)